api/discovery/v1alpha1/peeringrequest_types.go

```diff
@@ -53,7 +53,8 @@ type OriginClusterSets struct {
 
 // PeeringRequestStatus defines the observed state of PeeringRequest
 type PeeringRequestStatus struct {
-	AdvertisementStatus advtypes.AdvPhase `json:"advertisementStatus,omitempty"`
+	BroadcasterRef      *v1.ObjectReference `json:"broadcasterRef,omitempty"`
```

Comment:
you can use `DeploymentReference` struct ([here](https://github.com/liqotech/liqo/blob/master/pkg/object-references/types.go)) as suggested in the new version of kubernetes, which discourages the use of `ObjectReference`